src/lazy/expanded/macro_evaluator.rs

```diff
@@ -1302,6 +1302,36 @@ mod tests {
         )
     }
 
+    #[test]
+    fn read_system_eexp_with_delimited_tagged_arg_group() -> IonResult<()> {
+        // Empty delimited argument group
+        bin_stream_eq(
+            &[
+                0xEF, // System macro, address follows as 1-byte FixedUInt
+                0x03, // make_string
+                0x02, // Argument group
+                0x01, // FlexUInt 0: delimited group
+                0xF0, // Delimited END
+            ],
+            r#""" // <-- empty string "#,
```

Comment:
It took me a minute to figure out what "empty string" was referring to. This might make it easier for other readers.
```suggestion
            r#" "" // <-- empty string "#,
```